pipeline/__init__.py

```diff
@@ -1,11 +1,8 @@
 import os
-from pickle import load
 
-from typing import List
+from pipeline.objects.pipeline_v2 import PipelineV2
 
-from dill import loads, dumps
-
-# from pipeline import logging
+__all__ = ["PipelineV2"]
 
 
 CACHE_DIR = os.getenv("PIPELINE_CACHE_DIR", "./cache")
```

Comment:
An aside while we're overhauling stuff: we should consider conforming to the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) here. Specifically, using `$XDG_CACHE_HOME` as the base directory rather than `.`, falling back to `$HOME/.cache` if that environment variable is empty or not set.